tests/tests/bitasset_tests.cpp

```diff
@@ -930,6 +930,13 @@ BOOST_AUTO_TEST_CASE( hf_935_test )
    // need additional block to avoid prev: popping block would leave head block null error
    generate_block( skip );
 
+   // testnet needs a lot more blocks(35) here to pass
+   if(strcmp(GRAPHENE_SYMBOL, "TEST") == 0) {
+      for(int b = 0; b<35; b++)
+         generate_block( skip );
+   }
+
+
```

Comment:
Please undo this change, it's not the correct way to fix the issue.

Reply:
Please check: https://github.com/bitshares/bitshares-core/pull/1594/commits/b0c55b2bb829aafb6c7a5036492505e630052a34

Passing in mainnet and testnet. Following comments here: https://github.com/bitshares/bitshares-core/pull/1574#discussion_r257478235